operator/nft.go

```diff
@@ -208,3 +213,189 @@ func checkOrCreateWGAIngressChain(ctx context.Context, nft *nftables.Conn, table
 
 	return nil, nil
 }
+
+// dnsRule is equivalent to
```

Comment:
i dont really like this. this is significantly less readable and has no benefit.

Reply:
Yeah I was also unsure about this. It feels wrong to use exec for this stuff when we can directly talk with nftables but it's truly hard to read. I'll revert